src/ctap/hid/receive.rs

```diff
@@ -143,6 +143,7 @@ impl MessageAssembler {
 
     fn parse_init_packet(
         &mut self,
+        env: &impl Env,
```

Comment:
There's APIs like this that needs to access the constants but don't have env as parameter originally. This one can be modified to gain access to Env easily but I'm not sure if there's any other places that are more difficult to get Env.

Reply:
It shouldn't get harder to add than this.

FYI: In this particular instance actually, I was wondering if we can make the HID parser completely independent of CTAP. This constant is one of the very few dependencies left. But it's an independent question from this PR, so feel free to ignore :)

Reply:
+1 it's not a problem to add `&mut impl Env` where needed. Also in the future (for clock) we might need to make it `&mut E` and have `<E: Env>` on the function or class (like `CtapState`).

Reply:
I would still make this `&mut impl Env` instead of `&impl Env` even if we don't need it to be mutable. Because we don't have concurrent code and the environment is always threaded, so it's not an issue. And having it already mut would permit to not have to change the call-sites in tests when we'll need to access mutable parts of the environment (like debug printing).